pocs/utils/rs232.py

```diff
@@ -40,6 +40,7 @@ def __init__(self,
                  port=None,
                  baudrate=115200,
                  name=None,
+                 timeout=2.0,
```

Comment:
I was expecting to see that you'd removed most or all of these params, and replaced them with kwargs to be passed to the serial_for_url call, or to the apply_settings() method of the serial object.

Reply:
Just tried doing this and it will involve a number of other changes not specifically related to the mount. I think a #181 specific PR would be best.